code/modules/antagonists/clock_cult/ratvarian_cyborg_modules.dm

```diff
@@ -9,10 +9,11 @@
 
 /obj/item/clock_module/Initialize()
 	..()
-	var/datum/clockcult/scripture/S = new scripture_datum
-	name = S.name
-	desc = S.desc
-	icon_state = S.button_icon_state
+	if(scripture_datum)
+		var/datum/clockcult/scripture/S = new scripture_datum
```

Comment:
Interesting code... Should probably just make `scripture_datum` actually be declared as the correct type up in the obj definition and clean this up.